src/components/collection_vote_box/CollectionSubmitVoteBox.tsx

```diff
@@ -247,15 +249,15 @@ const CollectionSubmitVoteBox: React.FC<CollectionSubmitVoteBoxProps> = (props)
                     <CollectionVoteYesButton
                       onClick={async () => await updateVote(true, !!updateFinalVote)}
                       onError={(error: unknown) => onVoteError(error, !!updateFinalVote)}
-                      disabled={isVotingDisabled || isApprovalDisabled || isLoading || isElectionClosed}
+                      disabled={voteInProgress || isVotingDisabled || isApprovalDisabled || isLoading || isElectionClosed}
```

Comment:
Here and in the `CollectionVoteNoButton`, this fixes the problem where a slow API call can leave both buttons enabled after clicking either one. If the user does click the other button, there can be a conflict on the back end if the vote is a final vote. Instead, we disable when an API call is actively operating and reset that state after the API completes.